### DISCUSSION TITLE
Add example of how to work with nsc?

### DISCUSSION THREAD

**Discussion by zeromsi** [nats-io/nats.go#769] [General]
How to add operator, account , user with nsc?

**Comment by ripienaar** [comment]
The docs are here https://docs.nats.io/nats-tools/nsc

**Comment by zeromsi** [comment]
@ripienaar  , I read the docs, I need go client to create, update operator, account, user instead of cmd. Thanks

**Comment by ripienaar** [comment]
I see. This package can’t do that, you can to some extend use nsc as a package and out jwt package to make the tokens etc. 

We don’t really have a good code library to do all nicely in one place I think

**Comment by zeromsi** [comment]
@ripienaar Thanks man!
